backend/routes/PostManagement/PostFetch.py

```
@@ -0,0 +1,137 @@
+from fastapi import APIRouter, Depends, HTTPException, Query, Request
+from sqlalchemy.orm import Session
+from typing import Optional, List, Union
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)